python/dgl/distributed/graph_services.py

```diff
@@ -286,22 +288,53 @@ def sample_neighbors(g, nodes, fanout, edge_dir='in', prob=None, replace=False):
 
         For sampling without replacement, if fanout > the number of neighbors, all the
         neighbors are sampled. If fanout == -1, all neighbors are collected.
+    output_format : str, optional
+        The format of the sampled subgraph. A user can specify the output subgraph to be
+        stored in a homogeneous graph format or in a heterogeneous graph format. The valid
+        values are "homogeneous" and "heterogeneous". This argument is only valid if
+        the input graph is heterogeneous.
```

Comment:
Do we support `heterogeneous` format now?

Reply:
In the end of the function, I tried to convert a sampled subgraph into a heterogeneous graph. so it does support heterogeneous graph. It's kind of slow. The question is whether this is necessary.

Reply:
If there is no strong motivation, we should consider omit it in this release, especially if later we found a better heterograph solution.